### ISSUE TITLE
Naming

### ISSUE THREAD

**Issue by uloco** [uloco/numix-cursor#10]
Do you really want to change the name of the cursor from `numix-cursor` to `custom-cursors`? By using the theme name, we get more attraction from all the numix theme users I think. This doesn't mean I am totally against changing the name, but just interested in why you would want to change it. (BTW: How did you stumble upon my cursor at all?)

If you really do prefer another name, I think the name `custom cursors`, just needs an additional real name prefix then, rather than just `custom`. I think here of something like `locodott custom cursor`, (`locodott` = `uloco` + `osendott` :smiley:).

**Comment by osendott** [comment]
my apologies, I had changed it in the script while testing/working and my repo trying to avoid any confusion between it and your original theme. I don't really care what it's called lol also, in the testing repo, I have an updated script, fixed the dark/light theme stuff (i think). It's uncommented, and kinda ugly, but it should work. i'm getting ready to head out for a bit, but when I get back I'll fix all the name changes and stuff, or you can if you want, just don't forget to change them inside the tar.gz's themselves too.. I still haven't finished a proper menu, I keep getting side tracked. Also, if we use the Numix name, we may have to get permission first.

As to how I found them, I can't remember.. It was back in my ubuntu days I think. I believe I was looking for numix-icons on gnome-look and noticed your cursors, but I may be mistaken.

**Comment by osendott** [comment]
good idea keeping discussion on github incase others join, i'm not very familiar with how to message on here, so I have to go back and find previous conversations and add to them :) If you check the testing repo, i've got everything updated quite a bit. You will also notice I have added a file that will allow you to configure the colors of the script. I haven't messed with it much (i did manage to turn the background green lol) but if you play around with it, i'm sure you'll get it looking nice. I think there's also a note as to where the file should be placed and what it should be named. (it should be named .dialogrc and placed in the home directory)

also, i have already changed the script to cleanup if the user presses cancel or escape, that was one of my many merges over the weekend lol As to having to hit space in order to select the color, that's a unix thing and while I'm sure it would be possible to change it, it would take code that I couldn't even begin to imagine at this moment. I would assume you'd have to catch the keypress, if it's enter then you'd have to check which option is highlighted in the menu, which I'm not sure how, and ... well, there's a lot of logic operations involved lol ) anyways, there's a note in the readme about this, perhaps an onscreen prompt would work? "please highlight your desired color and press <space> to select" or something along those lines? I also commented the code in case there was any you didn't understand, but it's all pretty straight forward. I also combined all the color changes to create a dark theme into one statement, makes it a bit faster. I've also updated version to 0.9.9, but that's completely arbitrary and just so I can keep track of what i'm working on locally. Grab the updated script, see what you think. I've resized the dialogs and such. once you colorize it, it should be getting close to ready. I'm still looking at adding a review of the user choices before anything is done, but I messed up in the rewrite and instead of doing everything at the end, it processes as it goes, so that'll have to change if there is to be a review. what do you think? whew this was a lot to write too :)

**Comment by osendott** [comment]
upon further investigation, i'm looking into possibly using whiptail instead of dialog for the interface. supposedly it's installed on just about every linux distro by default, it looks a bit more modern, and doesn't appear to require a spacebar to select, etc. also looks like the syntax is almost identical so it won't be much to swap out one for the other. whiptail could be a bit more flexible as well. I'm reading up on it tonight, I'll try and get something thrown together tomorrow to test it out.

**Comment by uloco** [comment]
I'll have a look at this. Sounds good so far. 
For the conservation, you can just open a new issue on your testing repo or the original ones and I will see that since I followed you and the repositories ;)

**Comment by osendott** [comment]
also, found out that dialog also has a menu options, switched from the radiolist to menu, fixed the having to hit spacebar problem. as for the flickering, i think it's caused by how the script displays progress by flashing a dialog with "file" of "total files". not sure why it happens on some and not others. best fix would be to replace that part with a progress bar or --gauge as they're called (i believe). trying to figure that out.
